package.json

```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "@frctl/fractal": "^1.5.13",
     "@frctl/mandelbrot": "^1.10.1",
-    "@tacc/core-styles": "github:TACC/Core-Styles#75ea79c0",
+    "@tacc/core-styles": "github:TACC/Core-Styles#task/fp-1828-move-apcd-styles-to-core",
```

Comment:
Styles form APCD that can be used beyond CMS (https://github.com/TACC/Core-Styles/pull/56).